lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -70,51 +61,28 @@ BrowserEngine _detectBrowserEngine() {
   return detectBrowserEngineByVendorAgent(vendor, agent);
 }
 
-/// Detects samsung blink variants.
-///
-///  Example patterns:
-///    Note 2 : GT-N7100
-///    Note 3 : SM-N900T
-///    Tab 4 : SM-T330NU
-///    Galaxy S4: SHV-E330S
-///    Galaxy Note2: SHV-E250L
-///    Note: SAMSUNG-SGH-I717
-///    SPH/SCH are very old Palm models.
-bool _isSamsungBrowser(String agent) {
-  final RegExp exp = RegExp(
-      r'SAMSUNG|SGH-[I|N|T]|GT-[I|N]|SM-[A|N|P|T|Z]|SHV-E|SCH-[I|J|R|S]|SPH-L');
-  return exp.hasMatch(agent.toUpperCase());
-}
-
 /// Detects browser engine for a given vendor and agent string.
 ///
 /// Used for testing this library.
 @visibleForTesting
 BrowserEngine detectBrowserEngineByVendorAgent(String vendor, String agent) {
   if (vendor == 'Google Inc.') {
-    // Samsung browser is based on blink, check for variant.
-    if (_isSamsungBrowser(agent)) {
-      return BrowserEngine.samsung;
-    }
     return BrowserEngine.blink;
   } else if (vendor == 'Apple Computer, Inc.') {
     return BrowserEngine.webkit;
-  } else if (agent.contains('edge/')) {
-    return BrowserEngine.edge;
   } else if (agent.contains('Edg/')) {
     // Chromium based Microsoft Edge has `Edg` in the user-agent.
     // https://docs.microsoft.com/en-us/microsoft-edge/web-platform/user-agent-string
     return BrowserEngine.blink;
-  } else if (agent.contains('trident/7.0')) {
-    return BrowserEngine.ie11;
   } else if (vendor == '' && agent.contains('firefox')) {
     // An empty string means firefox:
     // https://developer.mozilla.org/en-US/docs/Web/API/Navigator/vendor
     return BrowserEngine.firefox;
   }
-  // Assume unknown otherwise, but issue a warning.
+
+  // Assume Blink otherwise, but issue a warning.
   print('WARNING: failed to detect current browser engine.');
```

Comment:
```suggestion
  print('WARNING: failed to detect current browser engine. Assuming it is `blink`.');
```

Reply:
Done, but since this is developer-visible message instead of "blink" I say "Chromium-compatible browser".